lib/rbac/approval_access.rb

```diff
@@ -0,0 +1,133 @@
+module RBAC
+  class ApprovalAccess < Access
+    include RBAC::Permissions
+
+    ADMIN_ROLE = 'Approval Administrator'.freeze
```

Comment:
@hsong-rh To make this generic we should keep these constants in your controller/mixin and call generic functions in here. This file shouldn't contain anything approval specific but rather generic function which can be fed in values.